### DISCUSSION TITLE
After build with vite, virtual tables do not have any rows until window resize

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#974] [Q&A]
Hi, 
I have a wired bug.
The table is working fine locally, but after build with vite it has no rows rendered on the live website.
When resizing the browser window the rows will appear.

"material-react-table": "^2.11.1",
Vite: 4.5.2

![image](https://github.com/KevinVandy/material-react-table/assets/17402724/ec285f14-cdca-40b2-a0f8-b799375782fe)

after resize:
![Screenshot 2024-02-05 162754](https://github.com/KevinVandy/material-react-table/assets/17402724/09acd60c-9f24-4989-ac53-017f123dd9df)

**Comment by KevinVandy** [comment]
Well that's not good. I tried to replicate the problem by building my own examples locally in vite, and then running vite serve, but cannot replicate the problem. 

Is this a recent bug for you? You could see if this happens while using MRT v2.1.0 to see if any of the recent performance optimizations have anything to do with it. If it still doesn't work back on that version, then I'm guessing there is something going on with your code.

**Comment by BennyAlex** [comment]
@KevinVandy 
hey with the latest versions we see no rows (even after resize) after build, locally works fine.
I assume one of the plugins I use cause the problem:

plugins: [
            dynamicImport(),
            pluginRewriteAll(),
            react(),
            svgr()
]

**Comment by BennyAlex** [comment]
I installed 2.5.2 and this works fine,
2.9.2 does not

**Reply by KevinVandy** [reply]
Share this behavior in a code sandbox or GitHub repo if you can
